From 14b10bcb2cb564ea0101bd9b8315c9d70ec56487 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Fri, 11 Dec 2020 11:14:31 +0800
Subject: [PATCH] drm/rockchip: vop2: Change > to >= for gt2/gt4 check

According to test: Cluster can work with >, But
Esmart must enable gt2 when src_h = 2 * dst_h;

Change-Id: I3347eb4e340df093f7ab29afcb3a80ae5e03b0cc
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 drivers/gpu/drm/rockchip/rockchip_drm_vop2.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c b/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
index cfd23e3ffafa..0a209a89be1a 100644
--- a/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
+++ b/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
@@ -1116,9 +1116,9 @@ static void vop2_setup_scale(struct vop2 *vop2, const struct vop2_win *win,
 	else
 		vscl_filter_mode = win_data->vsd_filter_mode;
 
-	if (src_h > (4 * dst_h))
+	if (src_h >= (4 * dst_h))
 		gt4 = 1;
-	else if (src_h > (2 * dst_h))
+	else if (src_h >= (2 * dst_h))
 		gt2 = 1;
 
 	if (gt4)
-- 
2.35.3

